jax/_src/public_test_util.py

```diff
@@ -226,11 +226,17 @@ def _merge_tolerance(tol, default):
   return out
 
 
-def check_jvp(f, f_jvp, args, atol=None, rtol=None, eps=EPS, err_msg=''):
+def check_jvp(f, f_jvp, args, atol=None, rtol=None, eps=EPS, err_msg='', constants=None):
   atol = _merge_tolerance(atol, default_gradient_tolerance)
   rtol = _merge_tolerance(rtol, default_gradient_tolerance)
   rng = np.random.RandomState(0)
   tangent = tree_map(partial(rand_like, rng), args)
+  # apply constants, if any, these should be args
+  # that we *do not* want to be random values for tangents, as that
+  # is impossible/incorrect (like a constant input arg)
+  if constants:
+    tangent = (tangent,) + constants
+    args = (args,) + constants
```

Comment:
I don't think `constants` is a good name for this. But also I think we shouldn't change `check_jvp` for one test case. Let's figure out another way to write the test? Can iterate in chat.